<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qt_listitemClass</class>
 <widget class="QWidget" name="qt_listitemClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1562</width>
    <height>796</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qt_listitem</string>
  </property>
  <widget class="QTreeView" name="treeView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1281</width>
     <height>781</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>1300</x>
     <y>0</y>
     <width>251</width>
     <height>301</height>
    </rect>
   </property>
   <property name="title">
    <string>商品筛选</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_need">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>(苹果|iphone|pro|pm|max)&amp;13</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>必须包含的内容</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>必须排除的内容</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_waste">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>苹果x&amp;iphone x&amp;mini&amp;特价&amp;置换&amp;仅自提&amp;【</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_save">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_timeDesc">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>时间降序</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_filterUncheck">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>过滤未认证</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_init">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>初始化</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_price_low">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_price_high">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_filterPrice">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>220</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>筛选</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_checkSQL">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>221</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>查看SQL</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>1300</x>
     <y>320</y>
     <width>251</width>
     <height>471</height>
    </rect>
   </property>
   <property name="title">
    <string>商品详情</string>
   </property>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>231</width>
      <height>281</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>229</width>
       <height>279</height>
      </rect>
     </property>
     <widget class="QTextEdit" name="textEdit_itemInfo">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>231</width>
        <height>281</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_qrcode">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qt_listitem.qrc"/>
 </resources>
 <connections/>
</ui>
